src/omnisharp-http-utils.el

```diff
@@ -0,0 +1,39 @@
+
+(defun omnisharp-get-host ()
```

Comment:
I think this could be an internal function, i.e. `omnisharp--get-host` with a two `--` prefix